ArtifactOntology.ttl

```diff
@@ -2441,7 +2442,7 @@ cco:LightingSystem rdf:type owl:Class ;
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/List
 cco:List rdf:type owl:Class ;
          rdfs:subClassOf cco:InformationBearingArtifact ;
-         cco:definition "An Information Bearing Artifact that represents an ordered sequence of values, where the same value may occur more than once."@en ;
+         cco:definition "An Information Bearing Artifact that contains an ordered sequence of values, where the same value may occur more than once."@en ;
```

Comment:
Following on the CodeList suggested revision: 

- List - An Information Bearing Artifact that consists of one or more Information Bearing Artifacts that carry Information Content Entities and are the subjects of some Sequence Position Ordinality.